### ISSUE TITLE
Wrong usage of aria-label attribute on upload button

### ISSUE THREAD

**Issue by tomasvn** [cheersmas/react-upload-box#10]
Path of the file `src/styles/components/Buttons/StyledUploadButton.tsx`

the `aria-label` should be on the button element or if you want to use it on `svg` than you need to specify it in `<title>Upload</title>` tags in svg

**Comment by cheersmas** [comment]
thanks for pointing it out. marking it close
